pkg/remote/remote.go

```diff
@@ -108,7 +108,7 @@ func (c *Client) clientFromURL(ctx context.Context, apiURL string) (*openai.Clie
 	env := "GPTSCRIPT_PROVIDER_" + env2.ToEnvLike(parsed.Hostname()) + "_API_KEY"
 	key := os.Getenv(env)
 
-	if key == "" {
+	if key == "" && !isLocalhost(apiURL) {
```

Comment:
Here, we no longer look for the credential in the store if the model is running locally. Not sure whether we care to support IPv6 (`::1`) here, but I can add that if we do.